Desktop_Interface/Labrador.pro

```diff
@@ -101,22 +98,38 @@ win32 {
     SOURCES += winusbdriver.cpp
     HEADERS += winusbdriver.h
 
+    RC_ICONS = build_win/appicon.ico
+
+    target.path = /
+    lib_deploy.path = /
+
+    firmware.path = /firmware
+    firmware.files = $$files(resources/firmware/*)
+
+    waveforms.path = /waveforms
+    waveforms.files = $$files(resources/waveforms/*)
+
+    INSTALLS += target lib_deploy firmware waveforms
+
     #libusbk include
     contains(QT_ARCH, i386) {
-        CONFIG(release, debug|release): LIBS += -L$$PWD/build_win/libusbk/bin/lib/x86/ -llibusbK
-        else:CONFIG(debug, debug|release): LIBS += -L$$PWD/build_win/libusbk/bin/lib/x86/ -llibusbK
```

Comment:
These two lines both point to the same library, and I realized later that maybe it was intended for one to link to the `.dll`, and the other to the `.lib`?